src/app/routes/api/suggestionHandler.ts

```diff
@@ -51,7 +51,8 @@ function sendSuggestion(requ){
 	};
 
 	log('Received comment from "%s"', email);
-	return new SuggestionModel(suggest).save();
+
+	return suggestionService.save(suggest);
```

Comment:
The HTTP response is still missing:
```
suggestionService.save(suggest).save().then(() => {
    okResponse(resp, { sent: true });
})
.catch(error => errorResponse(resp, error));
```
You need to declare `resp : Response` as well in the function signature then.